src/Mbstring/Mbstring.php

```diff
@@ -406,14 +406,34 @@ public static function mb_encoding_aliases($encoding)
 
     public static function mb_check_encoding($var = null, $encoding = null)
     {
+        if (PHP_VERSION_ID < 70200 && \is_array($var)) {
+            trigger_error('mb_check_encoding() expects parameter 1 to be string, array given');
```

Comment:
`, \E_USER_WARNING` as second arg (by default it's a notice)

Reply:
Thanks, I saw it and fixed it!